R/fdbk_asdataframe.R

```diff
@@ -1203,6 +1261,133 @@ fdbk_dt_binning <- function(DT,varToBin="level",binLower,binUpper){
 ########################################################################################################################
 ########################################################################################################################
 ########################################################################################################################
+#' Bin a data.table column into user defined bins and replace it with the bin center value.
```

Comment:
Could you please rerun document() to generate the help file (and example) for fdbk_dt_interpolate?